src/components/ticket-page/InnerTicketInfo.tsx

```diff
@@ -180,6 +184,12 @@ const InnerTicketInfo = (props: InnerTicketInfoProps) => {
     }
   };
 
+  const handleLocationDescriptionChange = async (newLocationDescription: string) => {
+    if (ticket.status == TicketStatus.PENDING || ticket.status == TicketStatus.open) {

```

Comment:
This should be `ticket.status === TicketStatus.OPEN` (capital OPEN) otherwise it won't build